synapse/handlers/message.py

```diff
@@ -62,6 +70,19 @@ def __init__(self, hs):
         self.storage = hs.get_storage()
         self.state_store = self.storage.state
         self._event_serializer = hs.get_event_client_serializer()
+        self._ephemeral_events_enabled = hs.config.enable_ephemeral_messages
+
+        # The scheduled call to self._expire_event. None if no call is currently
+        # scheduled.
+        self._scheduled_expiry = None  # type: Optional[IDelayedCall]
```

Comment:
```suggestion
        self._scheduled_expiry = None  # type: Optional[IDelayedCall]

```

just makes it a bit more readable